### PR TITLE
Create tumbling and patrollingUpDown blocks 

### PR DESCRIPTION
**Pull request by bethanyaconnor**
And update the levels that use them as custom blocks. This will allow us to translate those blocks as we support translating blocks from the block pool but not blocks defined in the `custom_blocks` field.

There are a couple of levels in the unreleased 2020 curriculum that also have this property but I don't want to edit them through a PR to avoid merge conflicts.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/blocks/GamelabJr/gamelab_patrollingUpDown.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "category": "SL1",
```
Comment (ajpal) [inline dashboard/config/blocks/GamelabJr/gamelab_patrollingUpDown.json:2]: What does this category mean?

File: dashboard/config/scripts/levels/Dance Party 1-validated.level [hunk 2]
```diff
@@ -55,13 +55,18 @@
...
...ight\",\r\n    \"blockText\": \"patrolling left and right\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"faceDirection\",\r\n    \"blockText\": \"{SPRITE} face {DIRECTION}\",\r\n    \"args\": [\r\n      {\r\n        \"name\": \"SPRITE\",\r\n        \"type\": \"Sprite\"\r\n      },\r\n      {\r\n        \"name\": \"DIRECTION\",\r\n        \"options\": [\r\n          [\r\n            \"right\",\r\n            \"1\"\r\n          ],\r\n          [\r\n            \"left\",\r\n            \"-1\"\r\n          ]\r\n        ]\r\n      }\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"followingSprite\",\r\n    \"blockText\": \"following {TARGET}\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ],\r\n    \"args\": [\r\n      {\r\n        \"name\": \"TARGET\",\r\n        \"type\": \"Sprite\"\r\n      }\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"wanderRewrite\",\r\n    \"blockText\": \"wandering (new)\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  }\r\n]",
```
Comment (ajpal) [inline dashboard/config/scripts/levels/Dance Party 1-validated.level]: It's a little hard to review from this diff but I'm assuming you just deleted these two custom blocks out of this level?
What are the rest of the custom blocks here? Can we delete them too?

File: dashboard/config/scripts/levels/Fish Tank 1-validated.level [hunk 3]
```diff
@@ -66,7 +67,7 @@
       <xml>
         <block type="when_run">
           <next>
-            <block type="gamelab_createNewSprite" inline="false">
```
Comment (ajpal) [inline dashboard/config/scripts/levels/Fish Tank 1-validated.level]: intentional?

File: dashboard/config/scripts/levels/Fish Tank 1-validated.level [hunk 4]
```diff
@@ -75,7 +76,7 @@
                 </block>
               </value>
               <next>
-                <block type="gamelab_addBehaviorSimple" inline="true">
```
Comment (ajpal) [inline dashboard/config/scripts/levels/Fish Tank 1-validated.level]: intentional?

File: dashboard/config/blocks/GamelabJr/gamelab_patrollingUpDown.json (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/config/blocks/GamelabJr/gamelab_patrollingUpDown.json:2]: No idea but it's the [gamelab_draggable](https://github.com/code-dot-org/code-dot-org/blob/219761a42ed9315f0eba5fb7f32475c2ce679f48/dashboard/config/blocks/GamelabJr/gamelab_draggable.json#L2) is in (I cloned both of these blocks from that one). Happy to change it if there's a better category.

File: dashboard/config/scripts/levels/Fish Tank 1-validated.level (hunk 3 above)
Comment (bethanyaconnor) [reply dashboard/config/scripts/levels/Fish Tank 1-validated.level]: weirdddd no it wasn't. I'll fix that...

File: dashboard/config/scripts/levels/Fish Tank 1-validated.level (hunk 4 above)
Comment (bethanyaconnor) [reply dashboard/config/scripts/levels/Fish Tank 1-validated.level]: Same here...don't know how this happened

File: dashboard/config/scripts/levels/Dance Party 1-validated.level (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/config/scripts/levels/Dance Party 1-validated.level]: I believe so. I'll delete them and double check

**Review by ajpal — APPROVED** [review]

**Pull request merged**